src/_component/about-me/about-me.pug

```diff
@@ -0,0 +1,26 @@
+.column(span="row")
+  .title-box
+    .title-wrapper
+      .first-title Photography
+      .second-title My passion
+.column(span="row" height="10")
+.column(span="row")
+  button.circle
+    a.video-link(href="https://www.youtube.com")
+      span.icon--video-camera
+.column(span="row" height="10")
+.column(span="row")
+  h1.another-title Why photography?
+.column(span="row")
+  .text-wrapper
+    p.paragraph Photography opens your mind and your eyes. You start noticing things in very high detail. Normal people wont notice a lot of things while walking on a street but you’ll start noticing a broken window, a out of colour rooftop, a structure that should not have been there. You travel, learn, make mistakes and then try to improve. Photography is not just a hobby, it is an art of studying people and places.
+.column(span="row" height="10")
+.column(span="row")
+  .button-wrapper
+    a.second-link(href="asset/doc/Resume-Bianca-Andrei.docx")
```

Comment:
Try to make a rule for yourself all the folders and all the files from your projects should !/not have capitalised letters because you can run into problems/error. Happen to me as well.

Reply:
You are right! I changed it :D

Reply:
You're not using $baseUrl